src/sync/sync_session.hpp

```diff
@@ -107,6 +107,8 @@ class SyncProgressNotifier {
     std::unordered_map<uint64_t, NotifierPackage> m_packages;
 };
 
+std::function<void(util::Optional<app::AppError>)> handle_refresh(std::shared_ptr<SyncSession>);
```

Comment:
Why is this a friend function? It seems like just a private member function?

Reply:
Yep, could be. I'll move it.